### PR TITLE
minor: move sleep to main travis so all commands get chance to output

### PR DESCRIPTION
**Pull request by rnveach**
Identified at https://github.com/checkstyle/checkstyle/pull/12391#issuecomment-1312514908,

Moves all sleep commands to travis so we don't have to add sleep to everything and everyone gets a chance for output.

I left sleep commands near "exit"s as I wasn't sure if travis would still execute the rest.

I also organized all sleep commands so they were similar in pattern to do a search and find.

### CODE REVIEW

**Comment by rnveach** [comment]
https://app.travis-ci.com/github/checkstyle/checkstyle/builds/257722125
Travis passes here with no issue. So it is unclear why the other PR failed in the first place.

**Comment by rnveach** [comment]
@romani @nrmancuso 
Travis failed in master and it gave a message, but a weird message.

https://app.travis-ci.com/github/checkstyle/checkstyle/jobs/588301324#L1729
> find: ‘/home/travis/.m2’: No such file or directory

I don't know how it could not be found. We have other jobs and this PR that pass. Dependencies are not being redownloaded, so they are being stored somewhere.

**Comment by rnveach** [comment]
Added mvn repository location display from https://stackoverflow.com/a/51235328/1016482 .

**Comment by rnveach** [comment]
Maven settings location in travis:
https://app.travis-ci.com/github/checkstyle/checkstyle/jobs/588304834#L383
````
[DEBUG] Reading global settings from /opt/mvn/conf/settings.xml
[DEBUG] Reading user settings from /home/travis/.m2/settings.xml
````

**Comment by rnveach** [comment]
https://app.travis-ci.com/github/checkstyle/checkstyle/jobs/588305018#L1168
Global settings exist, but local failed the build.
I wonder if the code at https://github.com/checkstyle/checkstyle/blob/79ca3a35ee7bddcecbe5907812e3332b72ba1162/.ci/travis.sh#L13-L25 is valid anymore, let me print it.

**Comment by rnveach** [comment]
https://app.travis-ci.com/github/checkstyle/checkstyle/jobs/588305243#L305
> Travis Home Located At:  /home/travis

Local settings is in travis home, but the file doesn't exist.

**Comment by rnveach** [comment]
I am stopping my analysis to move this forward to prevent more jobs from failing.

File: .ci/travis.sh [hunk 1]
```diff
@@ -145,7 +144,7 @@ quarterly-cache-cleanup)
   MVN_REPO=$(mvn -e --no-transfer-progress help:evaluate -Dexpression=settings.localRepository \
     -q -DforceStdout);
   if [[ -d ${MVN_REPO} ]]; then
-    find "$MVN_REPO" -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \;
+    find "$MVN_REPO" -maxdepth 4 -type d -mtime +90 -exec rm -rf {} || true;
```
[the remark this answers is not in the document]
Comment (nrmancuso) [reply .ci/travis.sh:147]: We should only append `|| true`, `-exec` will not behave correctly without termination of command (but not fail):

```

➜  checkstyle git:(sonarqube-update)     find notpresent -maxdepth 4 -type d -mtime +90 -exec rm -rf {} || true
 
find: missing argument to `-exec'
➜  checkstyle git:(sonarqube-update) echo $?
0



```

**Review by romani — APPROVED** [review]
Let's try no sleep in shells

File: .ci/travis.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (rnveach) [reply .ci/travis.sh:147]: Are you saying it should be
` find "$MVN_REPO" -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \; || true;`
?

**Comment by rnveach** [comment]
> Let's try no sleep in shells

I did leave some sleeps in place where they were that were close to an exit as I wasn't sure if travis would resume the after section or not.

File: .ci/travis.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (nrmancuso) [reply .ci/travis.sh:147]: > Are you saying it should be
> ` find "$MVN_REPO" -maxdepth 4 -type d -mtime +90 -exec rm -rf {} \; || true;`
> ?

Yes

[the remark this answers is not in the document]
Comment (nrmancuso) [reply .ci/travis.sh:147]: Final semicolon is optional

[the remark this answers is not in the document]
Comment (rnveach) [reply .ci/travis.sh]: Done.

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**